netlify.toml

```diff
@@ -861,3 +861,9 @@
 [[redirects]]
     from = "/docs/deployment/deploy-linode"
     to = "/docs/self-host"
+
+
+# Added: 2021-11-04
+[[redirects]]
```

Comment:
This came automatically but I'm not sure it is what we want

Reply:
It does seem you moved that page, correct? If that's the case then we do want this

Reply:
Ah but you added the index page. In which case could delete this

Reply:
jup just delete, we don't want this redirect

Reply:
The bot keeps adding them so I'm not sure how to do it